pam/integration_test.go

```diff
@@ -138,16 +138,16 @@ func TestPamSmAuthenticate(t *testing.T) {
 				ref := filepath.Join(cacheDir, db)
 				wants[db] = testutils.LoadAndUpdateFromGoldenDump(t, ref)
 
-				// TODO: make this better, using a reader
 				// Load temporary got to memory
-				tmp := filepath.Join(t.TempDir(), db+".dump")
-				f, err := os.Create(tmp)
+				f, err := os.Create(filepath.Join(t.TempDir(), db+".dump"))
```

Comment:
didn’t we say we would use an in memory buffer which can write an read from?

Reply:
Yeah, we did. Probably was a brain fry moment. My bad... 
(In my defense, we did say a lot of stuff yesterday 🥲).